environments/production/hiera/db.json

```diff
@@ -7,7 +7,7 @@
         "::oaeservice::backup::cassandra"
     ],
 
-    "duplicity::params::excludes": ["**/snapshots"],
+    "duplicity::params::excludes": ["**/snapshots","**/OpsCenter"],
```

Comment:
Just a dumb question from me:

The backup directory is `/data/cassandra/data`. The directory we want to exclude is `/data/cassandra/data/OpsCenter`. Will the `**/` glob work for that?